### ISSUE TITLE
Tablet for skype only for the older generation (people)

### ISSUE THREAD

**Issue by ymatesic** [mozilla/r2d2b2g#248]
In trying to get my parents up on skype and helping them understand the A-Z of turning on the computer, to hooking up to wireless internet (as is cheaper than ADSL) in Croatia, to getting them to understand the statistics of how much MB they have used, to buying credits and buying a monthly plan, to teaching them how to skype, make and receive calls, I learnt that this was just too complicated for them.  What the over 50 or 60 generation just need to communicate with their loved ones worldwide is a 7" tablet that runs just skype with video and voice and all the components that goes with a phone.  The phones now available for skype have only small screens (older people do not see all that well) and that is not suitable.  

So as a previous executive IT architect I researched possibilites and came across this article http://techcrunch.com/2008/07/21/the-techcrunch-web-tablet-project/ and http://www.siliconrepublic.com/digital-life/item/30323-first-firefox-os-smartphone and decided to leave a issue here.  

Is anyone willing to lead an effot to provide just a simple 7" tablet for skype/phone integration on firefox OS for under €50.  

Sure one can add all the other mobile phone apps (and install a more sophisticated interface with all the bells and whistles of a smart phone) however the older non computer literate and the older generation care nothing but for phone dialing, contact list, phone message retrieval, and skype ... and devices to support skype .... video camera and sound, they do not care even to get on the internet however the use of  firefox could stay as an add on option.  I am afraid that a simple packaged solution/option on a tablet will not be developed but that the manufacturers and telecom providers will put on bells and whistles that just are not needed for about 50% of the population making the device difficult for them to understand.

Can someone get back to me if they are doing something along these line .... coordinating such an effort as I am willing to test with my parents, people of their generation, non literate computer users, and people in old people's/retirement homes.

An interface not more complicated that the firefox OS similator as it now stands with the phone being skype. No more bells and whistles.  

Thanks for reading.  Keep the KIS principle in mind,

Kind regards to all the developers working on this project.  

Adrianna

**Comment by mykmelez** [comment]
Hi Adrianna,

That's an interesting idea, but the Simulator is intended to be a test environment for app developers, and this issue tracker only tracks enhancement requests related to testing apps. You might try posting your idea in the B2G discussion forum http://www.mozilla.org/about/forums/#dev-b2g, although in my experience you generally have to take on a project yourself to make it happen if you're the one who cares about it.

**Comment by ymatesic** [comment]
Thanks for your reply .... I emailed my original post to the forum
administrator.  Perhaps someone picks up on the fact that this could be a
big seller item.  We geeks sometimes forget that the majority of people
just want something simple. [?]

Even, I at 56, could no longer be bothered finding out all that my
smartphone can do.  Last thing I learned was the Dropbox app, so I
connected my computer and mobile and now I use it regularly.  But I came
upon it by chance.  I could not get my photos of my Sansumg galaxy ....
would not connect to any computer !  I visited computer shops, phone shops,
etc and no one could find out why I was having this problem.  Samsung
support was a joke.  Finally when testing whether a photocopy shop who had
latest computers could download my photos .... they told me about dropbox.
And voila my problem of transferring my photos fixed via a workaround.

Friends my age all complain about the smartphone and could not be bothered
with using what all is available.

However I notice the young have no trouble and all the time in the world to
investigate this device.

Sincerely,

Adrianna
